inventory/group_vars/all.yml

```diff
@@ -8,11 +7,1913 @@ dns_replace_address:
   - '127.0.0.1'
   - '0.0.0.0'
 
+dns_adblock_lists_defaults:
+  - name: easylist-adblock
+    type: adblock
+    url: https://v.firebog.net/hosts/Easylist.txt
+  - name: adaway
+    type: adblock
+    url: https://adaway.org/hosts.txt
+  - name: AdguardDNS
+    type: adblock
+    url: https://v.firebog.net/hosts/AdguardDNS.txt
+  - name: Admiral
+    type: adblock
+    url: https://v.firebog.net/hosts/Admiral.txt
+  - name: anudeepND
+    type: adblock
+    url: https://raw.githubusercontent.com/anudeepND/blacklist/master/adservers.txt
+  - name: yoyo
+    url: https://pgl.yoyo.org/adservers/serverlist.php?hostformat=hosts&showintro=0&mimetype=plaintext
+    type: adblock
+
 # List of extra domains to block, in addition to what we have listed
 # in `dns_adblock_lists`
 # Example:
 # dns_adblock_lists_custom:
 #   - type: adblock
 #     content: |
 #       ...
-dns_adblock_lists_custom: []
+dns_adblock_lists_custom:
```

Comment:
Would be awesome if we could have a comment on where these came from as well, for posterity.

Reply:
I'll verify exactly where they came from, but I suspect they're custom additions to help with 'generic' website adblocking via Support & Operations.